### DISCUSSION TITLE
Camel JMS component solace jNDi

### DISCUSSION THREAD

**Discussion by skumaravelan** [apache/camel-karavan#1381] [Q&A]
I would like to find out Solace configuration of Camel 
s components. I could not find the example that I can refer to use camel component.
Any lead would assist me.

**Comment by davsclaus** [comment]
duplicate of https://github.com/apache/camel-karavan/discussions/1381

**Comment by skumaravelan** [comment]
@davsclaus Both of them are closed?

**Comment by skumaravelan** [comment]
@davsclaus : could you point me the config that i can use.
